src/Mod/Path/PathScripts/PathToolBitLibraryGui.py

```diff
@@ -371,6 +375,55 @@ def __init__(self):
         self.setupUI()
         self.title = self.form.windowTitle()
 
+    def checkWorkingDir(self):
+        # users shouldn't use the example toolbits and libraries.
+        # working directory should be writable
+        PathLog.track()
+
+        workingdir = os.path.dirname(PathPreferences.lastPathToolLibrary())
+        defaultdir = os.path.dirname(PathPreferences.pathDefaultToolsPath())
+
+        dirOK = lambda : workingdir != defaultdir and (os.access(workingdir, os.W_OK))
+
+        if dirOK():
+            return True
+
+        qm = PySide.QtGui.QMessageBox
+        ret = qm.question(None,'', "Please set up Toolbit Working Directory", qm.Ok | qm.Cancel)
```

Comment:
Overall I really like the way this works, but this particular dialog stymied me for a moment: when it appeared I read it and thought "OK, but how do I _do_ that?" Of course, after clicking OK it was obvious that I was being walked through that process, but I wonder if there is a way to reword this dialog to be more like a question (whose answers are "OK" and "Cancel"). Something along the lines of:
```suggestion
        ret = qm.question(None,'', "Toolbit working directory not set up. Do that now?", qm.Ok | qm.Cancel)
```
(Side note - how does text like this get translated? Does this go in a `translate()` block?)